### PR TITLE
Add tooltip to repo icons in explore page

### PR DESCRIPTION
**Pull request by renbaoshuo**

### CODE REVIEW

**Comment by lunny** [comment]
Could you paste the screenshot?

**Comment by renbaoshuo** [comment]
Snipaste_2022-07-05_17-08-51

File: templates/explore/repo_list.tmpl [hunk 1]
```diff
@@ -32,9 +32,9 @@
 							{{end}}
 						{{end}}
 						{{if .IsFork}}
-							{{svg "octicon-repo-forked"}}
+							<i class="ui icon tooltip" data-content="{{$.locale.Tr "repo.fork"}}" data-position="bottom center">{{svg "octicon-repo-forked"}}</i>
```
Comment (silverwind) [inline templates/explore/repo_list.tmpl]:
```suggestion
							<span class="tooltip" data-content="{{$.locale.Tr "repo.fork"}}" data-position="bottom center">{{svg "octicon-repo-forked"}}</span>
```

File: templates/explore/repo_list.tmpl [hunk 2]
```diff
@@ -32,9 +32,9 @@
...
 						{{else if .IsMirror}}
-							{{svg "octicon-mirror"}}
+							<i class="ui icon tooltip" data-content="{{$.locale.Tr "mirror"}}" data-position="bottom center">{{svg "octicon-mirror"}}</i>
```
Comment (silverwind) [inline templates/explore/repo_list.tmpl]:
```suggestion
							<span class="tooltip" data-content="{{$.locale.Tr "mirror"}}" data-position="bottom center">{{svg "octicon-mirror"}}</span>
```

**Review by delvh — APPROVED** [review]

**Review by silverwind — APPROVED** [review]
Actually confirmed myself its working.

**Comment by 6543** [comment]
:rocket:

**Pull request merged**